app/views/layouts/application.html.erb

```diff
@@ -1,18 +1,25 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>JakSieMaszSlowfood</title>
     <% if Rails.env.test? %>
       <%= tag.meta id: :fake_position, content: fake_location %>
     <% end %>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>JakSieMaszSlowfood</title>
+
+    <link href="https://fonts.googleapis.com/css?family=Play" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway|Roboto" rel="stylesheet">
```

Comment:
You might as well add `Play` with this single line as you did `Raleway` and `Roboto` - Think you can do `Raleway|Roboto|Play` and it'll pull all 3 font families from google fonts

Reply:
Okay, I'll try to do that. 👍